features/command_line/bisect.feature

```diff
@@ -54,12 +54,12 @@ Feature: Bisect
       Bisect started using options: "--seed 1234"
       Running suite to find failures... (0.16755 seconds)
       Starting bisect with 1 failing example and 9 non-failing examples.
+      Checking that failures are order-dependent... failure is order-dependent
 
-      Round 1: searching for 5 non-failing examples (of 9) to ignore: .. (0.30166 seconds)
-      Round 2: searching for 3 non-failing examples (of 5) to ignore: .. (0.30306 seconds)
-      Round 3: searching for 2 non-failing examples (of 3) to ignore: .. (0.33292 seconds)
-      Round 4: searching for 1 non-failing example (of 2) to ignore: . (0.16476 seconds)
-      Round 5: searching for 1 non-failing example (of 1) to ignore: . (0.15329 seconds)
+      Bisecting over non-failing examples 1-9 .. ignoring examples 6-9
+      Bisecting over non-failing examples 1-5 .. ignoring examples 4-5
+      Bisecting over non-failing examples 1-3 .. ignoring example 3
+      Bisecting over non-failing examples 1-2 . ignoring example 1
```

Comment:
Yep, I'll look at putting that back. It dropped out because there wasn't a natural equivalent of `:bisect_round_finished` in the recursive scheme, but I'll work it back in.

There's an illustration of the more complex recursive case [here](https://gist.github.com/urbanautomaton/089df056dbe0b5575200) - in the case where both sides need recursing on it just doesn't display any ignorable ids at the moment. I'm thinking of adding text like "Both sides implicated - splitting..." to indicate this situation. Will follow up with a proper test to illustrate.